zingolib/src/wallet/transactions/recording.rs

```diff
@@ -21,33 +21,12 @@ use crate::{
         traits::{self, DomainWalletExt, Nullifier, Recipient},
     },
 };
-
-use super::TxMapAndMaybeTrees;
-impl TxMapAndMaybeTrees {
-    /// During reorgs, we need to remove all txns at a given height, and all spends that refer to any removed txns.
-    pub fn invalidate_all_transactions_after_or_at_height(&mut self, reorg_height: u64) {
-        let reorg_height = BlockHeight::from_u32(reorg_height as u32);
-
-        self.transaction_records_by_id
-            .invalidate_all_transactions_after_or_at_height(reorg_height);
-
-        if let Some(ref mut t) = self.witness_trees {
-            t.witness_tree_sapling
-                .truncate_removing_checkpoint(&(reorg_height - 1))
-                .expect("Infallible");
-            t.witness_tree_orchard
-                .truncate_removing_checkpoint(&(reorg_height - 1))
-                .expect("Infallible");
-            t.add_checkpoint(reorg_height - 1);
-        }
-    }
-
+impl super::TransactionRecordsById {
```

Comment:
please move this impl block to either transaction_records_by_id.rs or a submod of transaction_records_by_id

Reply:
Done.